pom.xml

```diff
@@ -31,11 +31,11 @@
     <parent>
         <groupId>io.gravitee</groupId>
         <artifactId>gravitee-parent</artifactId>
-        <version>22.3.0</version>
+        <version>23.5.0</version>
     </parent>
 
     <properties>
-        <gravitee-apim.version>4.6.8</gravitee-apim.version>
+        <gravitee-apim.version>4.9.0-alpha.2-SNAPSHOT</gravitee-apim.version>
```

Comment:
It seems to me that the idea is to not have SNAPSHOT dependency in a non-alpha release of a plugin. i'm right ?